src/test/java/org/programmers/springjpa/persistence/OrderPersistenceTest.java

```diff
@@ -0,0 +1,86 @@
+package org.programmers.springjpa.persistence;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.EntityManagerFactory;
+import jakarta.persistence.EntityTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.programmers.springjpa.domain.entity.Member;
+import org.programmers.springjpa.domain.entity.Order;
+import org.programmers.springjpa.domain.entity.OrderStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Transactional
+@SpringBootTest
```

Comment:
```suggestion
@DataJpaTest
@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

대체제로 사용할 수 있을 것 같습니다.